### PR TITLE
Helm chart standardization

### PR DESCRIPTION
**Pull request by blaskoa**
Preparation for first major release by introducing nice-to-have features:

- Common labels
- PDBs
- More consistent naming of objects
- Better configuration with the option to either provide helm values or bring your own configmap

### CODE REVIEW

File: templates/station-deployment.yaml [hunk 1]
```diff
@@ -1,30 +1,52 @@
...
+{{- $name := include "smartface.station.name" . -}}
+{{- $selectorLabels := include "smartface.station.selectorLabels" . -}}
 apiVersion: "apps/v1"
 kind: "Deployment"
 metadata:
-  name: {{ .Values.station.name | quote }}
+  name: {{ $name | quote }}
   labels:
-    app: {{ .Values.station.name | quote }}
+    {{- include "smartface.api.labels" . | nindent 4 }}
```
Comment (FooRider) [inline templates/station-deployment.yaml:9]: I believe there's a wrong value referenced here
```suggestion
    {{- include "smartface.station.labels" . | nindent 4 }}
```

Comment (blaskoa) [reply templates/station-deployment.yaml]: thank you, fixed

**Review by FooRider — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
 ### Added
+- Common labels with option to specify custom labels/annotation for objects
+- PodDisruptionBudgets for deployments that can be scaled
+- Support for custom KEDA triggers
+- Existing configmap for rabbitMQ and S3 is no longer necessary and the chart can create them from provided values, which is also the new default behavior for S3
 
+### Breaking change
+- Changed default behavior for creating S3 configuration. If you like to continue managing the previously created S3 config map please use the `configurations.s3.existingConfigMapName` field. Otherwise the ConfigMap will be managed by the helm chart using the values provided in `configurations.s3`
+  - The ConfigMap keys for existing config map are no longer configurable, so if you want to keep using the ConfigMap not managed by this chart then please make sure that the key match what the helm chart expects
+- TODO Deployments and service use more standard pod selectors - this should be a breaking change because deployment selectors are immutable
```
Comment (blaskoa) [inline CHANGELOG.md:14]: Resolve this TODO, also in README

**Comment by elcomtik** [comment]
I just tested this on my deployment, LGTM

**Pull request merged**